tests/test_cli.py

```diff
@@ -238,6 +294,36 @@ def test_split():
                 ecount += len(di)
 
 
+def test_split_correct_offset():
+    # Arrange
+    path_in = retrieve_data("rtdc_data_hdf5_rtfdc.zip")
+    path_out = path_in.with_name("split")
+    path_out_nc = path_in.with_name("split_nc")
+
+    path_out.mkdir()
+    path_out_nc.mkdir()
+
+    with h5py.File(path_in, 'r') as hf:
+        fl_offset = min(np.min(hf["events"]["trace"]["fl1_raw"][:]), 1)
+
+    # Act
+    paths = cli.split(path_in=path_in, path_out=path_out, split_events=3,
+                      ret_out_paths=True, correct_offset=True)
+    # do i need to time.sleep(1) here so the file names dont collide?
```

Comment:
I only used `time.sleep` in tests so that the files will have a different hash, because then the time stamps in the files are different.